com.musala.simple.smarthome/com/musala/simple/smarthome/devices/SmartBulb.java

```diff
@@ -0,0 +1,16 @@
+package com.musala.simple.smarthome.devices;
+
+import com.musala.simple.smarthome.ISmartDevice;
+
+/**
+ * @author yoan.petrushinov
+ *
```

Comment:
One more little change we need - every chass/interface's JavaDoc should give some information about the class, e.g. ISmartDevice is a backbone for all devices, while SmartBulb represents a bulb device and implements {@link ISmartDevice}. Please correct this.